src/token.rs

```diff
@@ -0,0 +1,27 @@
+use clap::ArgMatches;
+
+use crate::util::file::*;
+
+pub async fn token(matches: &ArgMatches)
+{
+    if let Some(token) = matches.get_one::<String>("set")
+    {
+        let mut path = cargo_path(std::env::current_dir().unwrap()).await.unwrap();
```

Comment:
Hmmm, should we do the current_dir? :thinking: Would this fail if they are in say `year_2022/src/bin/day_01/` and runs `cargo aoc token set xxx`. Then they move to the directory `year_2022` and try to run `cargo aoc run -d 1 -y 22`. Will it find `.env` and the token then?

Reply:
This is maybe just the API for my `cargo_path(path)` being a little bit jank. It puts the env file in the cargo_folder (the same folder as Cargo.toml is located). The argument to the function is just the starting search location. Would probably have been best for to just hard-code it in the function

Reply:
Ofc my bad🤦‍♂️ I didn't think of it finding the cargo root as I was thinking to much of current_dir. This should work!